src/Avalonia.Controls/Automation/Peers/TextBoxAutomationPeer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Avalonia.Automation.Provider;
+using Avalonia.Controls;
+
+#nullable enable
+
+namespace Avalonia.Automation.Peers
+{
+    public class TextBoxAutomationPeer : ControlAutomationPeer, IValueProvider
+    {
+        public TextBoxAutomationPeer(TextBox owner)
```

Comment:
Should this peer read out the placeholder/watermark textif the TextBox has no text entered?

Reply:
Yep, probably! It also needs to implement `ITextProvider` so a fair bit of work will need to be done on this class in a future PR.